MonkeyWrench.Web.UI/EditLane.aspx.cs

```diff
@@ -705,7 +706,16 @@ protected void cmdSave_Click (object sender, EventArgs e)
 		lane.enabled = chkEnabled.Checked;
 		lane.additional_roles = txtRoles.Text;
 		lane.priority = Int32.Parse (lstPriority.SelectedItem.Value);
-		lane.is_protected = chkProtected.Checked;
+		lane.is_protected = chkProtected.Checked;
+
+		// Don't ever pull more than 100 commits
+		try {
+			int commits = Int32.Parse (txtFetchCommits.Text);
+			lane.max_commits_to_fetch = commits >= 100 ? 100 : commits;

```

Comment:
When I mentioned `git bisect` I meant running that locally on your own machine instead of on a CI machine, so I don't think that will ever be implemented in Wrench :smile: